mlir/utils/jenkins/Jenkinsfile

```diff
@@ -506,6 +536,11 @@ pipeline {
                                         check_RockE2ETests_Navi3x(true)
                                     }
                                 }
+                                if ( "${CODEPATH}" == 'navi4x' ) {  
```

Comment:
I think we need an "else if" here, or navi3x could run in its section just above and then run again in the else just below.